src/Application/Kernel.php

```diff
@@ -54,30 +53,12 @@ private function initContainer(): Container
             ->addInitializer(new RouteBindingInitializer())
             ->addInitializer(new PDOInitializer());
 
-        return $container;
-    }
-
-    private function initDiscoveryLocations(): void
-    {
-        $this->discoverTempestNamespaces();
-        $this->discoverInstalledPackageLocations();
-    }
-
-    private function initConfig(Container $container): void
-    {
-        // Register AppConfig
-        $container->config($this->appConfig);
-
-        // Scan for package config files
-        foreach ($this->appConfig->discoveryLocations as $package) {
-            $configFiles = glob(path($package->path, 'Config/**.php'));
-
-            foreach ($configFiles as $configFile) {
-                $configFile = require $configFile;
-
-                $container->config($configFile);
-            }
+        /** @var class-string<\Tempest\Bootstraps\Bootstrap> $bootstrap */
+        foreach ($this->bootstraps as $bootstrap) {
+            $container->singleton($bootstrap, fn () => new $bootstrap($container));
```

Comment:
You're passing the container to every bootstrap class, but that's not defined in the interface.

Reply:
i didn't think constructors are typically defined in interfaces, but i can add it. Or we can go back to requiring it in the `boot()` signature.

@brendt - any preference?

Reply:
Yeah, i'm removing this loop, so if i understand correctly, i don't need to update the interface. 😃❓